src/ng/interpolate.js

```diff
@@ -283,27 +289,55 @@ function $InterpolateProvider() {
         }
       }
 
+      if (concat.length === 1 && expressionPositions.length === 1) {
```

Comment:
Can we remove the initial `singleExpression` assignment and change this to `singleExpression = (concat.length === 1 && expressionPositions.length === 1)`?